python/py-rchitect/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        randy3k rchitect 0.3.40 v
+name                py-rchitect
+revision            0
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Mapping R API to Python
+long_description    {*}${description}
+
```

Comment:
does this install any architecture-dependent files? If not you should set `supported_archs` and `platforms` accordingly.

Reply:
Yes, this port does.